docs/swagger.yaml

```diff
@@ -204,96 +213,205 @@ components:
       x-examples:
         example-1:
           imageId: laptop_image4433
+    CategoryEnum:
+      type: string
+      title: CategoryEnum
+      enum:
+        - General
```

Comment:
Shouldn't we use UPPER_CASE_SNAKE_CASE?

Reply:
Oh ok. Cant find it in the rfc - should we add it there?